Gemfile

```diff
@@ -1,3 +1,20 @@
+# frozen_string_literal: true
+
+active_record_version = ENV.fetch('ACTIVE_RECORD_VERSION', nil)
+
 source 'https://rubygems.org'
 
-gemspec
\ No newline at end of file
+gemspec
+
+gem 'activerecord', active_record_version || '>= 5.0.0'
```

Comment:
this has to be a direct dependency in the gemspec.. the below mysql2 gem version, we don't need to lock it, let it automatically resolve through gemspec dependent versions..

Reply:
done !